cli/replicasets/bootstrap_vshard.go

```diff
@@ -0,0 +1,57 @@
+package replicasets
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+
+	"github.com/tarantool/cartridge-cli/cli/common"
+)
+
+func BootstrapVshard(ctx *context.Ctx, args []string) error {
+	instancesConf, err := getInstancesConf(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to get instances configuration: %s", err)
+	}
+
+	conn, err := getControlConn(instancesConf, ctx, nil)
+	if err != nil {
+		return fmt.Errorf("Failed to connect to Tarantool instance: %s", err)
+	}
+
+	if err := bootstrapVshard(conn); err != nil {
+		return fmt.Errorf("failed to bootstrap vshard: %s", err)
+	}
+
+	log.Infof("Vshard is bootstrapped successfully")
+
+	return nil
+}
+
+func bootstrapVshard(conn net.Conn) error {
+	_, err := common.EvalTarantoolConn(conn, bootstrapVshardBody)
+	if err != nil {
+		if strings.Contains(err.Error(), `Sharding config is empty`) {
+			// XXX: see https://github.com/tarantool/cartridge/issues/1148
+			log.Warnf(
+				`It's possible that there is no running instances of some configured vshard groups. ` +
+					`In this case existing storages are bootstrapped, but Cartridge returns an error`,
+			)
+		}
+
+		return err
+	}
+
+	return nil
+}
+
+var (
+	bootstrapVshardBody = `
```

Comment:
Not sure but maybe it's reasonable to call "can_bootstrap_vshard" before (see https://github.com/tarantool/cartridge/blob/e4ecef6fb2176889e4ebcdb99d8a19cbee70c876/cartridge/vshard-utils.lua#L416)

Reply:
I was thinking about it, but `can_bootstrap_vshard` returns just `false` if there is no router or all groups are bootstrapped. But `bootstrap_vshard` returns an error. In fact, `can_bootstrap_vshard` is needed only for front-end to decide if `Bootstrap vshard` button should be drawn or not.

Reply:
Ok